openedx_webhooks/jira/tasks/update_latest_github_activity.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
This task *only* updates JIRA issues, it has no awareness of GitHub since it only receives static information.